### ISSUE TITLE
opendirSync is not implemented

### ISSUE THREAD

**Issue by 12people** [skpm/fs#30]
The readme states that "All the sync nodejs fs API is available". However, opendirSync seems to not have been implemented.

**Comment by mathieudutour** [comment]
Yeah it's a new method (added in Node 12 IIRC). Happy to accept a PR that implements it

**Comment by maheshgaur143** [comment]
writeFile and readFile is not implemented. ?
